package.json

```diff
@@ -37,58 +70,30 @@
     "stylelint-config-standard": "^17.0.0",
     "supertest": "^3.0.0",
     "testpilot-ga": "^0.3.0",
-    "uglifyify": "^4.0.3"
+    "uglifyify": "^4.0.3",
+    "webpack": "^3.4.1"
   },
   "engines": {
     "node": ">=8.0.0"
   },
   "homepage": "https://github.com/mozilla/send/",
   "license": "MPL-2.0",
   "repository": "mozilla/send",
-  "availableLanguages": [
-    "en-US",
-    "zh-TW",
-    "zh-CN",
-    "cs",
-    "fr",
-    "fy-NL",
-    "de",
-    "hu",
-    "it",
-    "ja",
-    "kab",
-    "ms",
-    "nb-NO",
-    "nn-NO",
-    "pt-PT",
-    "pt-BR",
-    "ru",
-    "sk",
-    "sl",
-    "dsb",
-    "hsb",
-    "es-CL",
-    "es-ES",
-    "es-MX",
-    "sv-SE",
-    "tr",
-    "cy"
-  ],
   "scripts": {
     "build": "npm-run-all build:*",
-    "build:upload": "browserify frontend/src/upload.js -g uglifyify -o public/upload.js",
-    "build:download": "browserify frontend/src/download.js -g uglifyify -o public/download.js",
-    "build:version": "node scripts/version",
     "build:l10n": "cp node_modules/l20n/dist/web/l20n.min.js public",
```

Comment:
Not sure if we'd need this npm script anymore, or if we could make some shared vendor bundle w/ **jquery** and **jquery-circle-progress** using webpack.